content/blog/2022/07/psp-migration-script.md

```diff
@@ -0,0 +1,303 @@
+---
+title: Migrate your PSPs to Kubewarden policies!
+authors:
+- José Guilherme Vanz
+date: 2022-07-13
+---
+
+As announced in past [blog](https://www.kubewarden.io/blog/2022/01/mutating-policy-behave-as-validating/) [posts](https://www.kubewarden.io/blog/2022/05/psp-migration-docs/),
+Kubewarden has 100% coverage of the deprecated, and soon removed,
+Kuberernetes PSPs. If everything goes as expected the PSPs will
+be removed in Kubernetes v1.25 due for release on 23rd August 2022.
+With this in mind, the Kubewarden team has written a script that leverages the
+migration tool written by people from [AppVia](https://github.com/appvia/psp-migration),
+to migrate PSP automatically. With this script the PSP will be migrated for the
```

Comment:
I think, we'd benefit from telling users, very briefly albeit, what AppVia's script does because we've modified it to suit our usecase.

Reply:
I just copy the same text from the docs. Let me know if you believe it is not enough.